### ISSUE TITLE
Multisite Compatible?

### ISSUE THREAD

**Issue by Dean-Wilson** [nystudio107/craft-vite#61]
### Question
Should there be any issues when using with Multisite? Have been happily using this, working great and then somewhere along the way where I migrated from single site to multisite, Vite no longer connects to my site. The build command runs successfully and everything looks great after that but if I try and `ddev exec npm run serve` I just get a few of these, the main one:

`https://sitename.ddev.site:3000/src/js/app.js net::ERR_CONNECTION_REFUSED`

Have tried destroying and restarting, checking that port is open for Vite, making sure mutagen is off (turned that on at one point and wanted to rule it out), made sure the build command worked. Not sure how multisite might mess with it, doesn't seem like it should but I've run out of ideas and I'm already half way through the project using it so support would be greatly appreciated. 

### Additional context
Running within DDEV. I believe my config is set up as per https://nystudio107.com/docs/vite/

**Comment by khalwat** [comment]
Yes, it should work fine with multi-site, and you can even use a multi-environment config in the `config/vite.php` should you need it.

My guess is that your issue here is on the DDEV end of things. Without knowing the details of how you have everything set up, I can't really help much.

Specifically, what exactly did you change when "migrat[ing] from single site to multisite"? Is it the same DDEV container? Is port 3000 open on that container?

**Comment by Dean-Wilson** [comment]
Thanks so much for the response. Yeah I figured as much. TLDR - fixed, had upgraded ddev and didn't see that `docker-compose.vite.yaml` was lost in the process as that file is git ignored.

Looking at the commits over the last few times I've worked on this, I did upgrade ddev (noticed craftcms was recently added as a project type to ddev). The only changes that made there were changing the type from php to craftcms in `.ddev/config.yaml`. Then I added the additional hostname for the new site in the same file. I tried reverting the project type, completely pruning everything from docker, re-imnporting a fresh db and no luck. With docker ps - indeed that port was not exposed on ddev-router. That's when I went through the setup steps another time and noticed `docker-compose.vite.yaml` was missing - it didn't show as a change as the ddev folder has it git ignored.
